src/components/ChatBeginingText.js

```diff
@@ -0,0 +1,60 @@
+import React, {memo} from 'react';
+import PropTypes from 'prop-types';
+import styles from '../styles/styles';
+import Text from './Text';
+import withLocalize, {withLocalizePropTypes} from './withLocalize';
+import compose from '../libs/compose';
+
+const propTypes = {
+
+    isDefaultChatRoom: PropTypes.bool,
+    chatUsers: PropTypes.arrayOf(PropTypes.object),
+    ...withLocalizePropTypes,
+
+};
+
+const defaultProps = {
+    chatUsers: [],
+    isDefaultChatRoom: false,
+};
+
+const ChatBeginingText = ({isDefaultChatRoom, chatUsers, translate}) => (
+    <Text style={[styles.mt3, styles.w70, styles.textAlignCenter]}>
+        <Text>
+            {isDefaultChatRoom ? `${translate('reportActionsView.beginingOfChatHistroyPrivate')} ` : `${translate('reportActionsView.beginingOfChatHistroy')} `}
+        </Text>
+        {isDefaultChatRoom
+            && (
+                <Text>
+                    {`${chatUsers?.[0]?.displayName} ${translate('reportActionsView.beginingOfChatHistroyPrivateSectionPart')}`}
+                </Text>
+            )}
+        {!isDefaultChatRoom
+            && (
+                <Text>
+                    {chatUsers.map(({displayName, pronouns}, index) => (
+                        <Text key={displayName}>
+                            <Text style={[styles.chatTextStyle]}>
+                                {displayName}
+                            </Text>
+                            {(pronouns !== undefined && pronouns !== '')
```

Comment:
This could `pronouns` only.

Reply:
@parasharrajat I did that because sometimes pronouns key is like '' that's why I will use loadash get

Reply:
<img width="374" alt="Screenshot 2021-09-26 at 4 45 14 PM" src="https://user-images.githubusercontent.com/31244700/134806272-44476cc1-d5b0-4ebb-ac39-82c078894c28.png">